### ISSUE TITLE
Could someone give clear explanation of .cfg like batch, subdivisions, max_batches. momentum, etc? 

### ISSUE THREAD

**Issue by git-sohib** [AlexeyAB/darknet#1462]
I sort of know what some of the terminologies mean in .cfg file, but for some other future learners, it would be good to know each of them right from here and how they can be modified accordingly. To put it simple, it should answer the questions like **what is it?** and **why is it?**
Here are all of them:

batch -->
subdivisions --> 
height -->
width -->
channels -->
momentum -->
decay -->
angle -->
saturation -->
exposure -->
hue -->

learning_rate -->
max_batches --> 
policy  -->
steps  -->
scales  -->

batch_normalize   -->
size  -->
stride  -->
pad  -->
filters  -->
activation  -->

mask -->
anchors   -->
classes -->
num -->
jitter  -->
ignore_thresh  -->
truth_thresh  -->
random -->

I do know most of them, but as I said it would be a great manual for all other new-comers to both deep learning and yolo. 
Anyone can give his/her explanation for any of them and we can collect them all to one answer. 
Let's do it!

**Comment by ledmonster** [comment]
:+1:

**Comment by AlexeyAB** [comment]
Read wiki:

https://github.com/AlexeyAB/darknet/wiki/CFG-Parameters-in-the-%5Bnet%5D-section

https://github.com/AlexeyAB/darknet/wiki/CFG-Parameters-in-the-different-layers

**Comment by ledmonster** [comment]
Thank you. I didn't aware of wiki.
